src/GraphQL/Execution/Processor.php

```diff
@@ -69,14 +70,14 @@ protected function getResolveFunction(AbstractField $field) {
   /**
    * {@inheritdoc}
    */
-  protected function resolveFieldValue(AbstractField $field, $contextValue, array $fields, array $args) {
+  protected function resolveFieldValue(FieldInterface $field, $contextValue, array $fields, array $args) {
     $resolveInfo = $this->createResolveInfo($field, $fields);
 
     if ($field instanceof Field) {
       if (($resolveFunction = $this->getResolveFunction($field)) && is_callable($resolveFunction)) {
         return $resolveFunction($contextValue, $args, $resolveInfo);
       }
-      else if ($propertyValue = TypeService::getPropertyValue($contextValue, $field->getName())) {
+      elseif ($propertyValue = TypeService::getPropertyValue($contextValue, $field->getName())) {
```

Comment:
Doesn't matter, just Drupal coding standards. The one important change is AbstractField to FieldInterface